odoo/addons/base/models/res_company.py

```diff
@@ -70,6 +70,7 @@ def _default_currency_id(self):
     website = fields.Char(related='partner_id.website', readonly=False)
     vat = fields.Char(related='partner_id.vat', string="Tax ID", readonly=False)
     company_registry = fields.Char(related='partner_id.company_registry', string="Company ID", readonly=False)
+    company_registry_placeholder = fields.Char(compute='_compute_company_registry_placeholder')
```

Comment:
Could this be just a related field?
```suggestion
    company_registry_placeholder = fields.Char(related='partner_id.company_registry_placeholder')
```

Reply:
You could overwrite it on partner. This way you avoid de-synchronizing the field value on partner and company.